js/repl/replUtils.js

```diff
@@ -155,6 +155,7 @@ export const persistedStateToEnvConfig = (
     node: envPresetDefaults.node.default,
     version: persistedState.version,
     builtIns: envPresetDefaults.builtIns.default,
+    corejs: "2",
```

Comment:
Why hard-coded?

Reply:
I changed it to do the same as the builtins option.